Podcast/DiscoverViewController.swift

```diff
@@ -26,32 +25,13 @@ class DiscoverViewController: ViewController, UITableViewDelegate, UITableViewDa
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .offWhite
-        
-        let tabbedPageViewController = TabbedPageViewController()
-        tabbedPageViewController.searchResultsDelegate = self
-        tabbedPageViewController.searchRequestsDelegate = self
-        let searchResultsController = tabbedPageViewController
-        searchController = UISearchController(searchResultsController: searchResultsController)
-        searchController.searchResultsUpdater = searchResultsController
-        
-        let cancelButtonAttributes: NSDictionary = [NSAttributedStringKey.foregroundColor: UIColor.sea]
-       UIBarButtonItem.appearance().setTitleTextAttributes(cancelButtonAttributes as? [NSAttributedStringKey: Any], for: .normal)
-        
-        searchController.searchBar.showsCancelButton = false
-        searchController.searchBar.searchBarStyle = .minimal
-        searchController.searchBar.placeholder = "Search"
-        navigationItem.titleView = searchController.searchBar
-        searchController.searchBar.sizeToFit()
-        
-        searchController.delegate = self
-        searchController.hidesNavigationBarDuringPresentation = false
-        searchController.dimsBackgroundDuringPresentation = false
-        definesPresentationContext = true
+        title = "Discover"
         
         tableView = UITableView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height), style: .grouped)
         for (contentClass, identifier) in zip(sectionContentClasses, sectionContentIndentifiers) {
             tableView.register(contentClass.self, forCellReuseIdentifier: identifier)
         }
+        
         tableView.backgroundColor = .paleGrey
         tableView.delegate = self
         tableView.dataSource = self
```

Comment:
don't forget to add "mainScrollView = tableView" for ViewController class to work.